pfcpiface/main.go

```diff
@@ -133,13 +35,11 @@ func main() {
 	// cmdline args
 	flag.Parse()
 
-	var (
-		conf Conf
-		fp   fastPath
-	)
-
-	// read and parse json startup file
-	ParseJSON(configPath, &conf)
+	// Read and parse json startup file.
+	conf, err := LoadConfigFile(*configPath)
+	if err != nil {
+		log.Fatalln("Error reading conf file:", err)
+	}
 
 	if level, err := log.ParseLevel(conf.LogLevel); err != nil {
```

Comment:
Since we validate config now and set log level to `info` by default, I suggest to remove this check and call just `log.SetLevel(conf.LogLevel)`

Reply:
In action: `FATA[2022-02-10T00:21:30Z]/pfcpiface/pfcpiface/main.go:41 main.main() Error reading conf file: not a valid logrus Level: "foo"`